src/Button.js

```diff
@@ -39,41 +48,46 @@ DialogButton.defaultProps = {
 
 DialogButton.displayName = "DialogButton";
 
-const styles = StyleSheet.create({
-  button: Platform.select({
-    ios: {
-      flex: 1,
-      justifyContent: "center",
-      alignItems: "center",
-    },
-    android: {
-      justifyContent: "center",
-      alignItems: "center",
-    },
-    web: {
-      justifyContent: "center",
-      alignItems: "center",
-    },
-  }),
-  text: Platform.select({
-    ios: {
-      textAlign: "center",
-      fontSize: 17,
-      backgroundColor: "transparent",
-    },
-    android: {
-      textAlign: "center",
-      backgroundColor: "transparent",
-      padding: 8,
-      fontSize: 14,
-    },
-    web: {
-      textAlign: "center",
-      backgroundColor: "transparent",
-      padding: 8,
-      fontSize: 14,
-    },
-  }),
-});
+const __styles = (isDark) =>
+  StyleSheet.create({
+    button: Platform.select({
+      ios: {
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+      },
+      android: {
+        justifyContent: "center",
+        alignItems: "center",
+      },
+      web: {
+        justifyContent: "center",
+        alignItems: "center",
+      },
+    }),
+    text: Platform.select({
+      ios: {
+        color: PlatformColor("link"),
+        textAlign: "center",
+        fontSize: 17,
+        backgroundColor: "transparent",
+      },
+      android: {
+        color: PlatformColor(
+          `@android:color/${isDark ? "link_text_dark" : "link_text_dark_light"}`
```

Comment:
So my understanding is that with `PlatformColor` on iOS we can automatically target the right color variant (depending on dark-mode being enabled or not), but on Android we still need to do the check ourselves, right?
Also, are the `link_text_dark` and `link_text_dark_light` colors set by default in all React Native apps?

Reply:
Well I'll speak like I'm an Apple pro 😂 iOS is more upscale 🤪
React native's documentation is poor it refers us to the colors of the OS, and Android does not seem to automatically manage the change of the color comparison iOS which managed very well

Reply:
Got it!